packages/table/src/common/_loading.scss

```diff
@@ -14,25 +14,39 @@
   }
 }
 
-.bp-table-cell,
-.bp-table-header {
-  &.pt-loading {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    cursor: default;
-
-    // Always set color to transparent so cell content can fade-in when transitioning from the
-    // loading state
-    // stylelint-disable-next-line declaration-no-important
-    color: transparent !important;
-    pointer-events: none;
-  }
+@mixin hide-text() {
+  // Always set color to transparent so cell content can fade-in when transitioning from the
+  // loading state
+  // stylelint-disable-next-line declaration-no-important
+  color: transparent !important;
+}
+
+@mixin disable-interaction() {
+  cursor: default;
+  pointer-events: none;
+}
+
+@mixin vertical-align() {
```

Comment:
thing is, `vertical-align` is an actual CSS property which this mixin has nothing to do with.

`vertical-center`?